mne/beamformer/tests/test_lcmv.py

```diff
@@ -666,46 +670,111 @@ def test_localization_bias_fixed(bias_params_fixed, reg, weight_norm, use_cov,
+        (0.00, 'max-power', 'unit-noise-gain', True, None, 35, 37, 0, 0),
+        (0.00, 'max-power', 'nai', True, None, 35, 37, 0, 0),
     ])
 def test_localization_bias_free(bias_params_free, reg, pick_ori, weight_norm,
-                                use_cov, depth, lower, upper):
+                                use_cov, depth, lower, upper,
+                                lower_ori, upper_ori):
     """Test localization bias for free-orientation LCMV."""
     evoked, fwd, noise_cov, data_cov, want = bias_params_free
     if not use_cov:
         evoked.pick_types(meg='grad')
         noise_cov = None
-    loc = apply_lcmv(evoked, make_lcmv(evoked.info, fwd, data_cov, reg,
-                                       noise_cov, pick_ori=pick_ori,
-                                       weight_norm=weight_norm,
-                                       depth=depth)).data
+    with pytest.warns(None):  # rank deficiency of data_cov
+        filters = make_lcmv(evoked.info, fwd, data_cov, reg,
+                            noise_cov, pick_ori=pick_ori,
+                            weight_norm=weight_norm,
+                            depth=depth)
+    loc = apply_lcmv(evoked, filters).data
+    if pick_ori == 'vector':
+        ori = loc.copy() / np.linalg.norm(loc, axis=1, keepdims=True)
+    else:
+        # doesn't make sense for pooled (None) or max-power (can't be all 3)
+        ori = None
     loc = np.linalg.norm(loc, axis=1) if pick_ori == 'vector' else np.abs(loc)
     # Compute the percentage of sources for which there is no loc bias:
-    perc = (want == np.argmax(loc, axis=0)).mean() * 100
+    max_idx = np.argmax(loc, axis=0)
+    perc = (want == max_idx).mean() * 100
+    assert lower <= perc <= upper
+    _assert_free_ori_match(ori, max_idx, lower_ori, upper_ori)
+
+
+# Changes here should be synced with the ones above, but these have meaningful
+# orientation values
+@pytest.mark.parametrize(
+    'reg, weight_norm, use_cov, depth, lower, upper, lower_ori, upper_ori', [
+        (0.05, 'unit-noise-gain-invariant', False, None, 38, 40, 0.52, 0.54),
+        (0.05, 'unit-noise-gain', False, None, 38, 40, 0.52, 0.54),
+        (0.05, 'nai', True, None, 56, 57, 0.56, 0.58),
+        (0.05, None, True, None, 27, 28, 0.54, 0.56),
+        (0.05, None, True, 0.8, 42, 43, 0.54, 0.56),
+        # no reg
+        (0.00, None, True, None, 50, 51, 0.57, 0.58),
+        (0.00, 'unit-noise-gain-invariant', True, None, 73, 75, 0.57, 0.58),
+        (0.00, 'unit-noise-gain', True, None, 73, 75, 0.57, 0.58),
+        (0.00, 'nai', True, None, 73, 75, 0.57, 0.58),
+    ])
+def test_orientation_max_power(bias_params_fixed, bias_params_free,
+                               reg, weight_norm, use_cov, depth, lower, upper,
+                               lower_ori, upper_ori):
+    """Test orientation selection for bias for max-power LCMV."""
+    # we simulate data for the fixed orientation foward and beamform using
```

Comment:
I have a neat `GitHub Actions` box saying `foward ==> forward`. Do you see those as well @larsoner or is that a new feature that is still being rolled out?